### ISSUE TITLE
log4j-cli.properties not found on Job Submitter

### ISSUE THREAD

**Issue by JSchulte01** [spotify/flink-on-k8s-operator#315]
When the Job Submitter runs the following error occurs
```
Job manager is ready now. Tried 5 time(s), every 5s and succeeded 3 time(s).

---------- Submitting job ----------
/opt/flink/bin/flink run --jobmanager jobmanager:8081 --parallelism 1 --detached --pyFiles /opt/flink/job --pyModule test

ERROR StatusLogger Unable to access file:/opt/flink/conf/log4j-cli.properties
 java.io.FileNotFoundException: /opt/flink/conf/log4j-cli.properties (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:90)
	at sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:188)
	at org.apache.logging.log4j.core.config.ConfigurationSource.fromUri(ConfigurationSource.java:321)
	at org.apache.logging.log4j.core.config.ConfigurationFactory$Factory.getConfiguration(ConfigurationFactory.java:443)
	at org.apache.logging.log4j.core.config.ConfigurationFactory$Factory.getConfiguration(ConfigurationFactory.java:385)
	at org.apache.logging.log4j.core.config.ConfigurationFactory.getConfiguration(ConfigurationFactory.java:293)
	at org.apache.logging.log4j.core.LoggerContext.reconfigure(LoggerContext.java:647)
	at org.apache.logging.log4j.core.LoggerContext.reconfigure(LoggerContext.java:668)
	at org.apache.logging.log4j.core.LoggerContext.start(LoggerContext.java:253)
	at org.apache.logging.log4j.core.impl.Log4jContextFactory.getContext(Log4jContextFactory.java:153)
	at org.apache.logging.log4j.core.impl.Log4jContextFactory.getContext(Log4jContextFactory.java:45)
	at org.apache.logging.log4j.LogManager.getContext(LogManager.java:194)
	at org.apache.logging.log4j.spi.AbstractLoggerAdapter.getContext(AbstractLoggerAdapter.java:138)
	at org.apache.logging.slf4j.Log4jLoggerFactory.getContext(Log4jLoggerFactory.java:45)
	at org.apache.logging.log4j.spi.AbstractLoggerAdapter.getLogger(AbstractLoggerAdapter.java:48)
	at org.apache.logging.slf4j.Log4jLoggerFactory.getLogger(Log4jLoggerFactory.java:30)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:329)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:349)
	at org.apache.flink.client.cli.CliFrontend.<clinit>(CliFrontend.java:89)
```

The ConfigMap generated by the controller includes a default `log4j-console.properties` but not a default `log4j-cli.properties`.

**Comment by JSchulte01** [comment]
Should be as simple as updating https://github.com/spotify/flink-on-k8s-operator/blob/master/controllers/flinkcluster/flinkcluster_converter.go#L1238 to add a default `log4j-cli.properties`

**Comment by regadas** [comment]
Closed via #316
